ballerina/tests/test.bal

```diff
@@ -0,0 +1,141 @@
+import ballerina/test;
+import ballerina/oauth2;
+// import ballerina/io;
+import ballerina/os;
+import ballerina/time;
+
+
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+configurable string refreshToken = ?; 
+
+configurable boolean isServerLocal = os:getEnv("IsServerLocal") == "true";
+
+OAuth2RefreshTokenGrantConfig auth = {
+       clientId: clientId,
+       clientSecret: clientSecret,
+       refreshToken: refreshToken,
+       credentialBearer: oauth2:POST_BODY_BEARER // this line should be added in to when you are going to create auth object.
+   };
+
+ConnectionConfig config = {auth : auth};
+final string serviceURL = isServerLocal ? "localhost:8080" : "https://api.hubapi.com";
+final Client hubspotClient = check new Client(config, serviceURL);
+
+// Change this value and test
+final int days = 40;
+
+
+string testEmailId = "";
+string cloneEmailId = "";
+
+@test:Config
+public function testCreateEmailEp() returns error? {
+    PublicEmail|error response = check hubspotClient->/marketing/v3/emails.post({
+        name: "Test email using API",
+        subject: "Marketing Email using API!"
+    });
+
+    if response is PublicEmail {
+        testEmailId = response.id;
+    } else {
+        test:assertFail("Failed to create new email");
+    }
+    
```

Comment:
Checking the response type would be sufficient in this case. Check similar places.

```suggestion
    test:assertTrue(response is PublicEmail);
```